build/images/client/Dockerfile

```
@@ -19,4 +19,9 @@ RUN porter mixin install kubernetes && \
     porter plugin install azure && \ 
     porter plugin install kubernetes
 
+# Copy the porter installation into a distroless container without root
+FROM gcr.io/distroless/static:debug
+COPY --from=builder /root/.porter /root/.porter
+ENV PATH "$PATH:/root/.porter"
+
 ENTRYPOINT ["porter"]
```

```suggestion
ENTRYPOINT ["/app/.porter/porter"]
```
Best practice to declare full path to entrypoint, even if executable is present in path.